packaging/debian/control

```diff
@@ -0,0 +1,22 @@
+Source: connman-curses-git
+Maintainer: Javier López <m@javier.io>
+Section: net
+Priority: optional
+Build-Depends: debhelper  (>=9), libjson-c2, libjson-c-dev, autoconf (>=2.69), automake, libncurses5, libncurses5-dev, libdbus-1-dev, dh-exec (>= 0.12)
```

Comment:
This project uses libjson0 (although I'm not aware of the difference between those two packages, they seem pretty close).